keras_cv/models/backbones/resnet_v1/resnet_v1_backbone_presets.py

```diff
@@ -0,0 +1,128 @@
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 4, 6, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "basic_block",
+        },
+    },
+    "resnet50": {
+        "metadata": {
+            "description": (
+                "ResNet model with 50 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 4, 6, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "block",
+        },
+    },
+    "resnet101": {
+        "metadata": {
+            "description": (
+                "ResNet model with 101 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 4, 23, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "block",
+        },
+    },
+    "resnet152": {
+        "metadata": {
+            "description": (
+                "ResNet model with 152 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 8, 36, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "block",
+        },
+    },
+}
+
+backbone_presets_with_weights = {
+    "resnet50_imagenet": {
+        "metadata": {
+            "description": (
+                "ResNet model with 50 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style). "
+                "Trained on ILSVRC 2012 (Imagenet) classification task."
+            ),
+        },
+        "config": backbone_presets_no_weights["resnet50"]["config"],
+        "weights_url": "https://storage.googleapis.com/keras-cv/models/resnet50/imagenet/classification-v0-notop.h5",
+        "weights_hash": "dc5f6d8f929c78d0fc192afecc67b11ac2166e9d8b9ef945742368ae254c07af",
+    },
+}
+
+backbone_presets = {
```

Comment:
Add these preset structs to [backbone_presets.py](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/backbones/backbone_presets.py).